ihmc-avatar-interfaces/src/main/java/us/ihmc/avatar/scs2/SCS2RobotRigidBodyMutator.java

```diff
@@ -72,9 +72,21 @@ private static class RigidBodyMutator
          massMutator.setAmplitude(defaultMass * DEFAULT_MASS_PERCENTAGE_MIN_MAX);
 
          comOffsetMutators = new YoFunctionGenerator[3];
-         comOffsetMutators[0] = new YoFunctionGenerator(YoGeometryNameTools.createXName(rigidBody.getName() + "_CoMOffset", "Mutator"), time, registry, false, dt);
-         comOffsetMutators[1] = new YoFunctionGenerator(YoGeometryNameTools.createYName(rigidBody.getName() + "_CoMOffset", "Mutator"), time, registry, false, dt);
-         comOffsetMutators[2] = new YoFunctionGenerator(YoGeometryNameTools.createZName(rigidBody.getName() + "_CoMOffset", "Mutator"), time, registry, false, dt);
+         comOffsetMutators[0] = new YoFunctionGenerator(YoGeometryNameTools.createXName(rigidBody.getName() + "_CoMOffset", "Mutator"),
+                                                        time,
+                                                        registry,
+                                                        false,
+                                                        dt);
+         comOffsetMutators[1] = new YoFunctionGenerator(YoGeometryNameTools.createYName(rigidBody.getName() + "_CoMOffset", "Mutator"),
+                                                        time,
+                                                        registry,
+                                                        false,
+                                                        dt);
+         comOffsetMutators[2] = new YoFunctionGenerator(YoGeometryNameTools.createZName(rigidBody.getName() + "_CoMOffset", "Mutator"),
+                                                        time,
+                                                        registry,
+                                                        false,
+                                                        dt);
```

Comment:
this formatting is ugly. personally I think sometime we're better than the formatter :P

Reply:
or you could extract the name part `YoGeometryNameTools.createXName(rigidBody.getName() + "_CoMOffset", "Mutator")` to local variables